RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -787,13 +801,13 @@ def runTauID(self):
                 mvaName_wGwGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_EC',
                 mvaName_woGwGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_BL',
                 mvaName_woGwGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_EC'
-            )
+            ))
```

Comment:
```suggestion
            ))
            _patTauDiscriminationByElectronRejectionMVA62018Raw = getattr(self.process,"patTauDiscriminationByElectronRejectionMVA62018Raw"+self.postfix)
```
for easier access later;
please apply this to everything that now has more than one `getattr`